SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -146,6 +146,8 @@ class SPATIALGDK_API USpatialNetDriver : public UIpNetDriver
 
 	Worker_EntityId WorkerEntityId = SpatialConstants::INVALID_ENTITY_ID;
 
+	TUniquePtr<SpatialVirtualWorkerTranslator> VirtualWorkerTranslator;
```

Comment:
If the Receiver needs the translator perhaps we can keep it private on NetDriver and pass it into the constructor of the Receiver?

Reply:
I think there's a much larger untangling that needs to be done. I'd be interested in digging into that a bit as part of the glue, but may need to wait for Alex's plan.